wiki/Installing-Oppia-Android.md

```diff
@@ -63,85 +341,183 @@ Please follow these steps to set up Oppia Android on your local machine.
-   - Under "**Gradle JDK**", select Android Studio Default JDK 11.
-   - Then, click "Apply" and "OK" to complete your setup.
-5. Click the elephant icon in the toolbar ("Sync Gradle") to ensure that all the correct dependencies are downloaded. (In general, you'll want to do this step any time you update your dependencies.)
-
 ## Run the app from Android Studio
 
-1. Go to Tools > AVD Manager, click "Create Virtual Device...". Then:
+1. Launch Android Studio, and on the welcome screen, select **Plugins** to install the Bazel for Android Studio plugin and restart Android Studio.
 
-   - Select a preferred device definition. In general, any device is fine, but you can use Pixel 3a as a default (if you're developing for phones) or Nexus 7 (if you're developing for tablets). After selecting a device, click "Next" at the bottom right to continue.
-   - Select a system image (in general, API Level 28, unless you're an M1 Mac user, in which case use API Level 29). Then click "Next".
-   - Click "Finish" to complete setup.
+2. On next open, click the options button next to the new project/open project to find the "Import Bazel Project" option.
 
-2. To run the app, select the emulator device you want from the dropdown menu to the left of the "Run" button in the toolbar.
+   <img width="1029" alt="Image" src="https://github.com/user-attachments/assets/efcc4e6e-b6ba-44fc-9c96-aa381de83911" />
 
-3. Finally, click the "Run" button.
+3. A setup wizard will open. For Workspace, click (**...**) to open file manager and select `/opensource/oppia-android` as the location since this is where we have our project's `WORKSPACE` file.
+
+   ![Image](https://github.com/user-attachments/assets/9baf6f71-560b-49fe-84a7-c7b846a64287)
+
+4. On **OK**, you will be asked to **Trust project**. Accept because we trust the project.
+
+   <img width="1029" alt="Image" src="https://github.com/user-attachments/assets/bfc91f6d-d809-418d-ab20-89d899f12f72" />
+
+5. On **next**, to Select Project View, **select Create from scratch**.
+
+   <img width="1029" alt="Image" src="https://github.com/user-attachments/assets/cdc110d6-2926-4f96-b7d8-117e2c716f76" />
+
+6. A file similar to the one below will be generated:
+
+   <img width="760" alt="Image" src="https://github.com/user-attachments/assets/5a26210f-865d-4048-b5a5-73379984bfb5" />
+
+7. In the above file, modify it so that it looks like below(copy and paste):
+   
+   ```
+   directories:
+      # Add the directories you want added as source here
+      # By default, we've added your entire workspace ('.')
+      .
+      
+   # Automatically includes all relevant targets under the 'directories' above
+   derive_targets_from_directories: false
+   
+   targets:
+   # If source code isn't resolving, add additional targets that compile it here
+      //:oppia_dev_binary
+      //app
+      //app/src/main/...
+      //domain/...
+      //model/...
+      //testing/...
+      //utility/...
+      //data/...
+      //scripts/...
```

Comment:
Probably too extensive. Let's do:

```
      //:oppia_dev_binary
      //app
      //app/src/main/...
      //domain/src/main/...
      //testing/src/main/...
      //utility/src/main/...
      //data/src/main/...
      //model/...
```

This drops scripts and all tests for a much faster sync.

Reply:
Updated